qiskit/utils/run_circuits.py

```diff
@@ -62,12 +62,9 @@ def find_regs_by_name(
     return found_reg
 
 
+# TODO: This function would be removed after Terra supports job with infinite circuits.
```

Comment:
Pretty curious about the pay-as-you-go hardware access model we were debating that spawned this TODO tbh.